src/user/user.c

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright 2020 Google LLC
- * Licensed under the Apache License, Version 2.0 (the "License");
+ * Licensed under the Apache License, Version 3.0 (the "License");
```

Comment:
mh?
There are no Apache 3 licenses https://www.apache.org/licenses/
And if there was, you should update all references to it (for example 4 lines below you link to the LICENSE-2.0)